### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by vmihalko**
This pull-request should fix all problems mentioned in #1.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,19 +10,39 @@
...
+
+Then you can formally verify RPM packages of your choice:
+
+```bash
+$ dnf download --source ${package} # get ${package}.src.rpm
+$ # or
+$ koji download-build src ${package} # get ${package}.src.rpm
+$ simple
+$ csmock -t ${tool} ${pkg-to-verify}.src.rpm
+$ or specific
```
Comment (kdudka) [inline README.md]: `or specific` should be excluded from the code block, so that it does not look like code to be executed.

File: README.md [hunk 2]
```diff
@@ -47,25 +67,31 @@ This is useful when you __do not__ want to run all tests in the `%check` section
                % (props.spec_in, strlist_to_shell_cmd(rpm_opts))
```
Comment (kdudka) [inline README.md:70]: Although not strictly, related to this pull request, we should replace https://github.com/csutils/csmock/blob/main/py/csmock#L283 with a persistent link.

**Review by kdudka — COMMENTED** [review]
Thank you for working on this!  Minor review comments inline.

File: README.md (hunk 1 above)
Comment (vmihalko) [reply README.md]: This comment should be resolved now.

File: README.md (hunk 2 above)
Comment (vmihalko) [reply README.md:70]: I agree, but what do you think about creating a proper `timeout` option for this?

Comment (kdudka) [reply README.md:70]: As discussed in person, such an option would certainly be useful!

Comment (vmihalko) [reply README.md:70]: https://github.com/aufover/experiments/pull/2/commits/d370d9fdc7571e3f067af0dc02417177da15ac11 is fix for this comment.

**Pull request merged**